### DISCUSSION TITLE
Blacklist typo

### DISCUSSION THREAD

**Discussion by Helvio88** [DblK/tinshop#34] [General]
`blacklist` is typo'ed as `backlist` at some parts of code and documentation.
Unfortunately, this could be a breaking change. :(

**Comment by DblK** [comment]
Good catch, will make a fix for it.
Yeah could be a breaking change but since it's not `v1.0` yet, I will do it!

Again thx

**Comment by DblK** [comment]
Done there https://github.com/DblK/tinshop/pull/37
